tests/scripts/build-release-downstream.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+set -ex
+
+# Load dot env file if available
+if [ -f .env ]; then
+    # shellcheck disable=SC2046
+    export $(grep -v '^#' .env | xargs -d '\n')
+fi
+
+MAKE='make --debug=v --output-sync'
+
+# build and push rook image
+$MAKE build BUILD_REGISTRY=local
+build_Image="local/ceph-amd64:latest"
+git_hash=$(git rev-parse --short "${GITHUB_SHA}")
+tag_Image=quay.io/ocs-dev/rook-ceph:v${BRANCH_NAME}-$git_hash
+docker tag "$build_Image" "$tag_Image"
```

Comment:
```suggestion
docker tag "$build_Image" "$tag_Image"
```
can we use podman?

Reply:
No we cant, as the make cmd creates the image with docker...

Reply:
in that use for `make bundle` only

Reply:
`make bundle` it running fine locally

Reply:
after discussing offline we agreed to use `podman push` with bundle and `docker push` for image